lib/twocohom.gi

```diff
@@ -1533,15 +1586,66 @@ local hom,mats,mode,m,min,i,j,mo,bas,a,l,ugens,gi,r,cy,act,k,it,p;
   until false;
 end);
 
+BindGlobal("RandomSubgroupNotIncluding",function(g,n,time)
+local best,i,u,v,start,cnt;
+  start:=Runtime();
+  best:=TrivialSubgroup(g);
+  cnt:=0;
+  while Runtime()-start<time or cnt<100 do
```

Comment:
Could you please use `GET_TIMER_FROM_ReproducibleBehaviour` so that the code results in reproducible behavior when requested?

Reply:
Done, Thanks.